aws-ssm-parameter/aws-ssm-parameter.json

```diff
@@ -5,105 +5,75 @@
     "properties": {
         "Type": {
             "type": "string",
-            "description": "The type of the parameter.",
-            "enum": [
-                "String",
-                "StringList",
-                "SecureString"
-            ]
-        },
-        "Value": {
-            "type": "string",
-            "description": "The value associated with the parameter.",
-            "minLength": 1,
-            "maxLength": 32768
+            "description": "The type of parameter."
         },
         "Description": {
             "type": "string",
-            "description": "The information about the parameter.",
-            "minLength": 0,
-            "maxLength": 1024
+            "description": "Information about the parameter."
         },
         "Policies": {
             "type": "string",
```

Comment:
[CFN Doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-parameter.html#cfn-ssm-parameter-policies) says `Policies` is of type `String`

Reply:
Tried creating it via CLI with Json value instead of String, and faced below error.

Parameter validation failed:
Invalid type for parameter Policies